ROMFS/px4fmu_common/init.d/airframes/1103_standard_vtol_sih.hil

```diff
@@ -54,7 +54,7 @@ param set-default CA_SV_CS1_TRQ_P 1
 param set-default CA_SV_CS1_TYPE 3
 param set-default CA_SV_CS2_TRQ_Y 1
 
-param set HIL_ACT_REV 32
+param set HIL_ACT_REV 16
```

Comment:
Why is there at all a reversion needed in simulation? Can't we change the model?

Reply:
Same comment here

Reply:
addressed in latest commit & comment :)